packages/selected-network-controller/src/SelectedNetworkController.ts

```diff
@@ -189,16 +240,31 @@ export class SelectedNetworkController extends BaseController<
         },
       );
     }
+    // Update the network client ID for the specified domain
```

Comment:
sorry, another nit. I think the method name is self describing

Reply:
removed here: https://github.com/MetaMask/core/pull/3908/commits/ecb5e72a77608459747c279a2e14aad4c94b63a1